bouncer/tests/broker_fee_collection_test.ts

```diff
@@ -189,14 +189,18 @@ async function testBrokerFees(asset: Asset, seed?: string): Promise<void> {
   const balanceBeforeWithdrawalBigInt = BigInt(
     amountToFineAmount(balanceBeforeWithdrawal, assetDecimals[asset]),
   );
-  const minExpectedBalanceAfterWithdrawal =
-    balanceBeforeWithdrawalBigInt + earnedBrokerFeesAfter - maxWithdrawalFee[asset];
   const detectWithdrawalGasFee =
     balanceBeforeWithdrawalBigInt + earnedBrokerFeesAfter - balanceAfterWithdrawalBigInt;
+  if (['FLIP', 'DOT', 'USDC'].includes(asset.toString())) {
```

Comment:
?

Reply:
Not clear to me how this related to the polkadot version

Reply:
It's a typo, should be flip, eth, usdc... I added this to debug a bouncer issue, I'll fix the typo. 

The bouncer issue is that the fee broker fee estimation can return different values depending on which order they run in.